2024/11/4/cancellable-timeout/cancellable-timeout.ts

```
@@ -0,0 +1,11 @@
+export default function setCancellableTimeout(
+  callback: Function,
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)